packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -0,0 +1,321 @@
+        <label for="fileUploadWithAttachmentsAndValidation" class="FileUploaderInput__label">Label</label>
+        <div class="FileUploaderInput__dropZone">
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#upload" />
+          </svg>
+          <label for="fileUploadWithAttachmentsAndValidation" class="FileUploaderInput__dropZoneLabel">
+            <span class="FileUploaderInput__link link-primary link-underlined">Upload your file</span>
+            <span class="FileUploaderInput__dragAndDropLabel">or drag and drop here</span>
+          </label>
+          <div class="FileUploaderInput__helperText">Max file size is 10 MB</div>
+          <input type="file" id="fileUploadWithAttachmentsAndValidation" class="FileUploaderInput__input" />
+        </div>
+        <div class="FileUploaderInput__message">Danger validation message</div>
+      </div>
+      <!-- FileUploaderInput: end -->
+
+      <ul class="FileUploader__list">
+        <li>
+
+          <!-- FileUploaderAttachment: start -->
+          <div class="FileUploaderAttachment">
+            <svg width="24" height="24" aria-hidden="true">
+              <use xlink:href="/icons/svg/sprite.svg#file" />
+            </svg>
+            <div class="text-truncate">My resume.docx</div>
+            <button type="button" class="FileUploaderAttachment__remove">
+              <span class="accessibility-hidden">Delete</span>
+              <svg width="24" height="24" aria-hidden="true">
+                <use xlink:href="/icons/svg/sprite.svg#close" />
+              </svg>
+            </button>
+          </div>
+          <!-- FileUploaderAttachment: end -->
+
+        </li>
+        <li>
+
+          <!-- FileUploaderAttachment: start -->
+          <div class="FileUploaderAttachment">
+            <svg width="24" height="24" aria-hidden="true">
+              <use xlink:href="/icons/svg/sprite.svg#file" />
+            </svg>
+            <div class="text-truncate">
+              My resume with a name that is too long so it needs to be trimmed. You're not gonna see this part!.pdf
+            </div>
+            <button type="button" class="FileUploaderAttachment__remove">
+              <span class="accessibility-hidden">Delete</span>
+              <svg width="24" height="24" aria-hidden="true">
+                <use xlink:href="/icons/svg/sprite.svg#close" />
+              </svg>
+            </button>
+          </div>
+          <!-- FileUploaderAttachment: end -->
+
+        </li>
+      </ul>
+
+    </div>
+    <!-- FileUploader: end -->
+
+  </div>
+
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Fluid Width</h2>
+
+  <!-- FileUploader: start -->
+  <div class="FileUploader FileUploader--fluid">
+
+    <!-- FileUploaderInput: start -->
+    <div class="FileUploaderInput FileUploaderInput--fluid has-drag-and-drop">
```

Comment:
`FileUploaderInput--fluid` do we need this class?

Reply:
No, good point.